attachments.scad

```diff
@@ -1631,7 +1627,7 @@ module edge_mask(edges=EDGES_ALL, except=[]) {
 //   corners = Corners to mask.  See [Specifying Corners](attachments.scad#subsection-specifying-corners).  Default: All corners.
 //   except = Corners to explicitly NOT mask.  See [Specifying Corners](attachments.scad#subsection-specifying-corners).  Default: No corners.
 // Side Effects:
-//   Tags the children with "remove" (and hence sets `$tag`) if no tag is already set.
```

Comment:
This side effect description is still correct, and arguably clearer to the end user.